.github/workflows/crates-release.yml

```diff
@@ -0,0 +1,30 @@
+# See https://crates.io/docs/trusted-publishing
+name: Publish to crates.io
+on:
+  workflow_dispatch:
```

Comment:
I think we'll want to automate this, similar to https://github.com/bootc-dev/containers-image-proxy-rs/blob/b5e0861ad5065f47eaf9cda0d48da3529cc1bc43/.github/workflows/cratesio-release.yml

Reply:
Yeah agree, I think we can auto-publish the crates on release tags.

Reply:
Updated, and now only publish two crates: `bootc-internal-utils` and `bootc-internal-blockdev`